appeals/api/src/database/seed/data-test.js

```diff
@@ -654,6 +654,8 @@ export async function seedTestData(databaseConnector) {
 					status = 'invalid';
 				}
 
+				const source = Math.random() < 0.5 ? 'back_office' : 'front_office';
```

Comment:
I am sorry, but what's the reasoning behind the randomizer here?

Reply:
I guess I was thinking roughly 50% of them could be FO and the others BO?

Reply:
I didn't want to assign them linearly because status is assigned linearly. So all the awaiting reviews would be FO and all the invalids would be BO. Assigning them linearly within each status seemed overkill